### DISCUSSION TITLE
r10.net

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#23628] [Q&A]
`https://www.r10.net/`
https://i.imgur.com/gaW8HrZ.png

The ad on the left and the pop-up ad shown on the first visit. Can these two ad types be hidden using scriptlet?
If the user manually closes these two ads, the site creates two cookies. These are "modalad10 10" and "r10LeftHide 1" so I think it can be blocked using scriptlet.

**Comment by stephenhawk8054** [comment]
Enable `tr: AdGuard Turkish` list and test again

**Comment by ghost** [comment]
@stephenhawk8054 I don't use the Turkish filter because I don't need it. I just need the user rule for these two ads.

**Reply by stephenhawk8054** [reply]
Then you can look up `r10.net` in https://filters.adtidy.org/extension/ublock/filters/13.txt and see which ones apply to it.
